controllers/controller_shared_test.go

```diff
@@ -296,6 +296,9 @@ var _ = Describe("GetMatchingInstances functions", Ordered, func() {
 	unreadyGrafana.Name = "unready-instance"
 
 	ctx := context.Background()
+	testLog := logf.FromContext(ctx).WithSink(logf.NullLogSink{})
+	logf.IntoContext(ctx, testLog)
```

Comment:
`context.Context` is immutable so you'll need to use the return value here:

```suggestion
	ctx = logf.IntoContext(ctx, testLog)
```

same for all other instances of this

Reply:
At least that is a very quick `sed` fix :)